src/components/fields/location-field/location-modal/location-search/location-search.styles.ts

```diff
@@ -15,7 +15,8 @@ export const SearchWrapper = styled.div<ISinglePanelStyle>`
 	flex: 1;
 	padding: 2rem 1.5rem 1rem;
 
-	${V2_MediaQuery.MaxWidth.tablet}, (orientation: landscape) and (max-height: ${V2_MediaWidths.mobileL}px) {
+	${MediaQuery.MaxWidth.lg}, (orientation: landscape) and (max-height: ${({ theme }) =>
+		Breakpoint["md-min"]({ theme })}px) {
```

Comment:
as this is a styled component, you can set the token directly like this (styled components will call the function automatically with the theme object)
```suggestion
	${MediaQuery.MaxWidth.lg}, (orientation: landscape) and (max-height: ${Breakpoint["md-min"]}px) {
```